dsm/resource_app.go

```diff
@@ -44,6 +45,21 @@ func resourceApp() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+            "mod_group": {
+                Type: schema.TypeMap,
+                Optional: true,
+                Elem: &schema.Schema{
+                    Type: schema.TypeString,
+                    Optional: true,
+                },
+            },
+            "del_group": {
```

Comment:
let us try if we can remove group from list of other_group

Reply:
Made the changes accordingly, user can delete from other_group itself.